chat-client/src/client/chat.test.ts

```diff
@@ -65,26 +66,41 @@ describe('Chat', () => {
         global.ResizeObserver = undefined
     })
 
-    it('publishes ready event and initial tab add event, when initialized', () => {
-        assert.callCount(clientApi.postMessage, 4)
+    it('publishes ready event when initialized', () => {
+        assert.calledWithExactly(clientApi.postMessage.firstCall, { command: READY_NOTIFICATION_METHOD })
+    })
 
-        assert.calledWithExactly(clientApi.postMessage.firstCall, {
+    it('creates initial tab when chat options are provided', () => {
+        const bannerText = 'This is a test banner message'
+        const eventParams = {
+            command: CHAT_OPTIONS,
+            params: {
+                chatNotifications: {
+                    bannerText: bannerText,
+                },
+            },
+        }
+        const sendToPromptEvent = createInboundEvent(eventParams)
+        window.dispatchEvent(sendToPromptEvent)
+
+        assert.calledWithExactly(clientApi.postMessage.firstCall, { command: READY_NOTIFICATION_METHOD })
```

Comment:
Do we send `onReady` event again, the 2nd time after the `CHAT_OPTIONS`? If no, we should remove this check from here, because it's confusing.